PlayerControls/sources/AirPlayView.swift

```diff
@@ -53,3 +59,12 @@ public final class AirPlayView: UIView {
         volumeView.setRouteButtonImage(props.icons.selected, for: .selected)
     }
 }
+
+class AirPlayMock: UIButton {
+    
+    var showsVolumeSlider: Bool = false
```

Comment:
Where this variable is used?

Reply:
It's a mock variable. If we won't add this one to `AirPlayMock` class, there will be a need to add another check in `AwakeFromNib`:
```
public override func awakeFromNib() {
          super.awakeFromNib()
          volumeView.showsVolumeSlider = false
}
```
`AirplayMock` class will be initialized only when the project is launched on Simulator, replacing `MPVolumeView`. Same as `showVolumeSlider`.